src/main/java/com/faforever/client/game/CustomGamesController.java

```diff
@@ -252,4 +268,23 @@ protected void onHide() {
     // Hide all games to free up memory
     filteredItems.setPredicate(game -> false);
   }
+
+  public void toggleSidePane() {
+    boolean currentlyShowingSidePane = preferencesService.getPreferences().isShowGameDetailsSidePane();
+    preferencesService.getPreferences().setShowGameDetailsSidePane(!currentlyShowingSidePane);
+    preferencesService.storeInBackground();
+
+    setSidePane(!currentlyShowingSidePane);
+  }
+
+  private void setSidePane(boolean displayed) {
+    if (displayed) {
+      toggleSidePaneButton.setText(i18n.get("view.hideSidePane"));
+      gamesGridPane.getColumnConstraints().stream().filter(c -> c.getHgrow() == Priority.SOMETIMES).forEach(c -> c.setMinWidth(300));
```

Comment:
please reference the column you want to set to 0 by id not by Hgrow.... Imagine somebody adds a column with Priority.Sometimes then he will wonder by it always collapses on HideSidePane

Reply:
oh, those column constraints are listed by column, that's bad design

Reply:
????